app/assets/stylesheets/vendor/_select2.scss

```diff
@@ -0,0 +1,535 @@
+.select2-container {
```

Comment:
File app/assets/stylesheets/select2.css should be removed since we have this one.